docs/install/installEverest.md

```diff
@@ -29,7 +29,134 @@ Percona Everest assists with installing all the necessary operators and required
+    
+    ```sh
+    mv everestctl-darwin-amd64 everestctl
+    ```
+
+6. Modify the file permissions:
+
+    ```sh
+    chmod +x everestctl
+    ```
+    
+7. From the installation wizard, provision and register the Kubernetes cluster in Everest using the following commands. Everest will search for the kubeconfig file in the `~/.kube/config` path. If your file is located elsewhere, set the `KUBECONFIG` environment variable before running the command:
+  
+    ```sh
+    export KUBECONFIG=~/.kube/config
+    ```
+
+  
+    ```sh
+    ./everestctl install operators
+    ```
+    
+    This will install all needed components in a namespace called `percona-everest`.
+
+    ??? example "Example"
+            
+            ? Everest URL http://127.0.0.1:8080
+            ? Choose your Kubernetes Cluster name k3d-dbaas-tilt
+            ? Do you want to enable monitoring? Yes
+            ? PMM URL Endpoint http://127.0.0.1
+            ? Username admin
+            ? Password *****
+            ? Name for the new monitoring instance my-pmm
+            ? Do you want to enable backups? No
+            ? What operators do you want to install? MySQL, MongoDB, PostgreSQL
+            
+
+    Alternatively, you can provision and register the Kubernetes cluster by running the installation in headless mode:
+        
+    ```
+    ./everestctl install operators --backup.enable=false --everest.endpoint=http://127.0.0.1:8080 --operator.mongodb=true --operator.postgresql=true --operator.xtradb-cluster=true --monitoring.enable=true --monitoring.type=pmm --monitoring.new-instance-name=my-pmm --monitoring.pmm.endpoint=http://127.0.0.1 --monitoring.pmm.username=admin --monitoring.pmm.password=admin  --skip-wizard
+    ```
+
+    #### Limitations
+        
+    * If the Everest CLI fails to install the operators, do the following:
+        
+        * [Uninstall Percona Everest](uninstallEverest.md).
+        
+        * Install Percona Everest, starting with the second step.
+
+    * If you install an operator after the initial provisioning and want to create a database cluster, restart the `everest-operator pod` by running the following commands:
+
+        ```sh 
+        kubectl -n percona-everest get po
+        kubectl -n percona-everest delete pod everest-operator-pod-name
+        ```
+
+        This will fix the database cluster creation. The issue will be fixed in the upcoming releases.
+
+    * If you don't enable monitoring during this provisioning step then you won't be able to enable it from the UI later. Make sure to fill in the monitoring details in the wizard.
+
+    * If you are using a PMM server instance with a self-signed certificate you cannot use HTTPS in the PMM URL endpoint.
+
+8. Go to [http://127.0.0.1:8080](http://127.0.0.1:8080) to open the Everest UI and create your first database cluster.
```

Comment:
Here, we should also say that the user should go to the external IP address he got on step 4, 127.0.0.1 is just an example.